lib/inspec/control_eval_context.rb

```diff
@@ -66,9 +66,15 @@ def method_missing(method_name, *arguments, &block)
             # We still haven't called it, so do so now.
             send(method_name, *arguments, &block)
           else
-            # If we couldn't find a plugin to match, maybe something up above has it,
-            # or maybe it is just a unknown method error.
-            super
+            begin
+              id = method_name
+              require "inspec/resources/#{id}"
+              klass = Inspec::Resource.registry[id.to_s]
+              klass.new(inspec, id, *arguments)
+            rescue LoadError
+              # TODO: aws
```

Comment:
If you're referring to the aws resources in lib/resources, those are deprecated and are scheduled for removal in 5.0 - so I'm not sure it's worth adding code to handle.

Reply:
If I understand the change correctly, we'll need code to handle AWS until we completely remove it otherwise those resources won't be loaded and users would be forced to use inspec-aws.

Reply:
I needed to handle it to make tests pass, so... done.